### DISCUSSION TITLE
[ZK Stack] - <Question about the decentralization prover>

### DISCUSSION THREAD

**Discussion by SuccinctPaul** [zkSync-Community-Hub/zksync-developers#311] [ZK Stack] [answered]
### Module Affected

- [ ] System contracts (era-system-contracts)
- [ ] Smart contracts (era-contracts)
- [ ] zkEVM (era-zk_evm)
- [ ] Server (zksync-era)
- [X] Prover (era-boojum)
- [ ] Fee withdrawer (era-fee-withdrawer)
- [ ] Block Explorer (clock-explorer)
- [ ] Portal - Wallet + Bridge (dapp-portal)
- [ ] Other (please specify below)

### Rust Version

1.74.0-nightly

### Operating System

- [X] Ubuntu
- [ ] MacOS
- [ ] Windows
- [ ] Other (Please specify below)

### Issue Description

Does zkStack support the decentralization prover?

### Expected Behavior

decentralization prover

### Current Behavior

.

### Prior Research

- [X] Yes, I searched for similar issues and did not find any matching ones.

**Comment by muradfelix** [comment]
dfdsds

**Comment by dutterbutter** [comment, marked as answer]
Not at this time but decentralization is apart of the roadmap.

**Comment by crimuntilve** [comment]
Yes, zkSync's zkStack does support decentralized provers. Decentralization of the prover is a key feature of zkSync's architecture, aimed at enhancing security, reliability, and scalability of the system. By distributing the computational workload across multiple provers, zkSync achieves greater resilience against attacks and ensures that no single point of failure exists in the system. This decentralized approach also enables zkSync to scale more effectively as demand for transaction processing increases.

**Comment by mganlok** [comment]
The zkStack currently does not support decentralized prover functionality.

**Comment by mancinottimoe** [comment]
ZKSync does support decentralized proof generation through its prover infrastructure. This allows for increased scalability, security, and decentralization of the network.

**Comment by segheamulchoidiaz** [comment]
Based on the limited information provided, it is unclear what the issue is. The question seems to be asking whether zkStack supports the decentralization prover, but no additional details or context are provided. Without further information, it is difficult to provide a more detailed answer.
